packages/components/src/components/primitives/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import {
+  Accordion as ChakraAccordion,
+  AccordionItem as ChakraAccordionItem,
+  AccordionButton as ChakraAccordionButton,
+  AccordionPanel as ChakraAccordionPanel,
+  AccordionIcon as ChakraAccordionIcon,
+} from '@chakra-ui/react';
+
+import {
+  AccordionProps,
+  AccordionStaticMembers,
+  AccordionItemProps,
+  AccordionButtonProps,
+  AccordionPanelProps,
+} from './types';
+
+export const Accordion: FC<AccordionProps> & AccordionStaticMembers = (
+  props
+) => {
+  return <ChakraAccordion {...props} />;
+};
+
+const AccordionItem: FC<AccordionItemProps> = (props) => (
+  <ChakraAccordionItem {...props} />
+);
+const AccordionButton: FC<AccordionButtonProps> = (props) => (
+  <ChakraAccordionButton {...props} />
+);
+const AccordionPanel: FC<AccordionPanelProps> = (props) => (
+  <ChakraAccordionPanel {...props} />
+);
```

Comment:
this components are redundant, you can safely remove then and use the original chakra component below

Reply:
done!